adoc/release-2.0.adoc

```diff
@@ -0,0 +1,136 @@
+* The `values.yaml` configuration file has undergone significant updates, including changes to the format and name of keys. When migrating from {product} 1.5.2 to {version}, the existing `values.yaml` configuration file (for example, `scf-config-values.yaml`) for 1.5.2 **cannot** be reused for {version}. A new `values.yaml` file (for example, `kubecf-config-values.yaml`) **must** be created and used instead. Refer to <<app.sample-values>> as a guideline for format and names valid for {version}
+* Fixed `appVersion` field in Chart yaml(s) to reflect the application version
+* Removed `sle12` and `cflinuxfs2` stack, which were preceded by warnings in CAP 1.5.x releases
+* Removed `cf-usb` service broker. Users should use Minibroker or the OSBAPI-compliant brokers provided by the public cloud platforms instead
+* `nfs-broker` is not included in the current release
+* `pxc` is no longer directly taken from upstream; we rely on a docker image (0.9.4) that uses version 5.7.28-31.41
+* Includes these {cf} component versions:
+** app-autoscaler: 3.0.0
+** bits-services: 2.28.0
+** bosh-dns-aliases: 0.0.3
+** bpm: 1.1.7
+** capi: 1.91.0
+** cf-acceptance-tests: 0.0.13
+** cf-deployment: 12.36
+** cf-smoke-tests: 40.0.128
+** cf-syslog-drain: 10.2.11
+** cflinuxfs3: 0.167.0
+** credhub: 2.5.11
+** diego: 2.44.0
+** eirini: 0.0.27 
+** garden-runc: 1.19.10
+** loggregator: 106.3.8
+** loggregator-agent: 5.3.7
+** log-cache: 2.6.8
+** nats: 33
+** postgres-release: 39
+** routing: 0.198.0
+** scf-helper: 1.0.13
+** silk: 2.28.0
+** sle15: 10.93
+** statsd-injector: 1.11.15
+** sync-integration-tests: 0.0.3
+** uaa: 74.15.0
+* Buildpacks:
+** binary-buildpack: 1.0.36
+** dotnetcore-buildpack: 2.3.9
+** go-buildpack: 1.9.11
+** java-buildpack: 4.29.1
+** nginx-buildpack: 1.1.7
+** nodejs-buildpack: 1.7.17
+** php-buildpack: 4.4.12
+** python-buildpack: 1.7.12
+** staticfile-buildpack: 1.5.5
+** ruby-buildpack: 1.8.15
+
+[id='sec.2_0.issue']
+==== Known Issues
+
+[IMPORTANT]
+====
+* The transition from {product} 1.5.2 to {product} 2.0 involves a migration of data rather than a direct upgrade. The procedure can be found at https://documentation.suse.com/suse-cap/2.0/single-html/cap-guides/#sec-cap-update 
+
+====
+
+[IMPORTANT]
+====
+* Autoscaler can go into a `CrashLoopBackoff` state if DNS setup is not complete by the time autoscaler comes up. To avoid this situation it is recommended the DNS entries are set up as soon as services (e.g. `router-public`) have external IPs assigned to them. The `asactors` pod can also be deleted to recover from this state as kubernetes will re-create the pod
+
+====
+
+[IMPORTANT]
+====
+* Autoscaler can go into a `CrashLoopBackoff` state when enabled using `helm upgrade` on systems where the DNS entries have already been set up. The `asactors` pod can be deleted to recover from this state as kubernetes will re-create the pod
+
+====
+
+* Eirini will only work on a cluster that has `.cluster.local` as the cluster domain
```

Comment:
What does this mean? Eirini works with LB

Reply:
This was listed in the Jira for release notes collection. I believe it's referring to `--cluster-domain=<default-local-domain>` described at https://kubernetes.io/docs/tasks/administer-cluster/dns-custom-nameservers/#introduction. This doesn't affect public clouds since that should already be the default. Will expand to clarify.